app/services/usps_in_person_proofing/proofer.rb

```diff
@@ -230,5 +231,9 @@ def dedupe_facilities(facilities)
         [facility.address, facility.city, facility.state, facility.zip_code_5]
       end
     end
+
+    def sort_by_ascending_distance(facilities)
+      facilities.sort_by { |f| f[:distance].delete_suffix(' mi').to_f }
```

Comment:
Turns out Ruby's `String#to_f` is super liberal, and is happy to parse a number out of a mixed string, maybe we drop the `.delete_suffix`?

```
 "5.111 mi".to_f
=> 5.111
```

```suggestion
      facilities.sort_by { |f| f[:distance].to_f }
```
and that way if USPS ever changes units or formatting, this will continue to work

Reply:
Super liberal, thanks for the knowledge! 🥳